diagnostic.md

```diff
@@ -100,11 +134,29 @@ object.
 ```
 
 ```sh
-# your answer here
+curl "http://www.example.com/donuts/42" \
+--include \
+--request PATCH \
+--header "Content-Type: application/json" \
+--data '{
+  "name": "German Cruller",
+  "price": "$0.99"
+}
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function () {
+  return $.ajax({
+    url: app.host + '/example/' + '6',
```

Comment:
Don't need this to be `app.host + /example/ + 6`, since "example" is the host in this situation